app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/ImportLoginsViewModel.kt

```diff
@@ -18,14 +22,17 @@ import javax.inject.Inject
 @Suppress("TooManyFunctions")
 @HiltViewModel
 class ImportLoginsViewModel @Inject constructor(
+    savedStateHandle: SavedStateHandle,
     private val vaultRepository: VaultRepository,
+    private val snackbarRelayManager: SnackbarRelayManager,
 ) :
     BaseViewModel<ImportLoginsState, ImportLoginsEvent, ImportLoginsAction>(
         initialState = ImportLoginsState(
-            null,
+            dialogState = null,
             viewState = ImportLoginsState.ViewState.InitialContent,
             isVaultSyncing = false,
             showBottomSheet = false,
+            snackbarRelay = ImportLoginsArgs(savedStateHandle).snackBarRelay,
```

Comment:
Do we need to pass this in as an argument? Could the screen just know to send the correct enum value?

Reply:
it worked like that cause of the "most recent" relay thing, which had the nullable "relay parameter". I can re-add that, I took the suggestion to move to enums to make it so it was always explicit.